bql/planner/planner_test.go

```diff
@@ -39,6 +39,8 @@ const (
 		/u<peter> "bought"@[2016-02-01T00:00:00-08:00] /c<model s>
 		/u<peter> "bought"@[2016-03-01T00:00:00-08:00] /c<model x>
 		/u<peter> "bought"@[2016-04-01T00:00:00-08:00] /c<model y>
+		/u<paul> "bought"@[2016-01-01T00:00:00-08:00] /c<model n>
```

Comment:
Thank you for the comment Kloss. I have just done so adding the following text to the description of the commit there:

> Two new triples were added so we could have, for both predicate and object positions, examples on which two triples have the same latest anchor (expecting 2 rows for `FILTER latest` in this case, in the place of only 1 as it was in the usual case tested so far).
> For that the triples added were the one with predicate `"bought"@[2016-04-01T00:00:00-08:00]` and the other with object `"turned"@[2016-04-01T00:00:00-08:00]`.
>
> The third triple was added so that both `/u<peter>` and `/u<paul>` would have in common two temporal predicates - this way we can test if only one `FILTER latest(?p)` is working as expected for multiple graph clauses inside of WHERE (if they share that same binding `?p`).
> For this, the triple added was the one with predicate `"bought"@[2016-01-01T00:00:00-08:00]`.